### DISCUSSION TITLE
Create List button stuck in loading state

### DISCUSSION THREAD

**Discussion by import-fola** [community/community#84201] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Anytime I tap the create list button to create a list, it stays in a loading state with the button greyed out and copy stating 'Creating...' . When I dismiss it I can see the list is actually created.
<img width="453" alt="Screenshot 2024-01-05 at 09 40 03" src="https://github.com/community/community/assets/19846552/799c2d61-d613-4054-be7a-33b463c110e9">

**Comment by BasicAcid** [comment]
See:
https://github.com/orgs/community/discussions/76972

**Reply by Miqueas** [reply]
This still happens to me on Edge (Linux and Windows)

**Comment by HyunJay52** [comment]
same thing happened to me 😵

**Comment by PrashantRatanparikhe** [comment]
It sounds like you're encountering an issue with a GitHub feature or web interface. If you're experiencing a problem with GitHub, there are a few steps you can take to troubleshoot or report the issue:

**Try a Different Browser:**
Switch to a different web browser to see if the issue persists. This can help identify whether the problem is specific to the browser you're currently using.

**Disable Browser Extensions:**
Some browser extensions might interfere with GitHub's functionality. Try disabling browser extensions one by one to see if any of them are causing the problem.

**Check GitHub Status:**
Before reporting an issue, check the [GitHub Status](https://www.githubstatus.com/) page to ensure there are no ongoing incidents or outages. If there is an incident, GitHub may already be aware of the problem and working on a solution.

**Contact GitHub Support:**
If the issue persists and you cannot find a resolution, you can contact GitHub support for assistance. Visit the GitHub Support page to submit a request.

**Comment by oliverthedeveloper** [comment]
Faults in Vivaldi - too!

**Comment by Jamim** [comment]
This discussion duplicates #50880, as a dozen of other reports.

**Comment by shocker990** [comment]
Claro que si

**Comment by ELhanysadaoui** [comment]
@Han

**Comment by Pinkolinka11** [comment]
https://open.spotify.com/track/4enbO9cZhJLSZ6Tf21LRnv?si=fdY55tAuTjSdFhcs_KJJ_Q%0A

Dne so 27. 4. 2024 18:07 uživatel ELhanysadaoui ***@***.***>
napsal:

**Reply by shocker990** [reply]
Pasamela

**Comment by Pinkolinka11** [comment]
Bez do kundy ty mrdko

Dne so 27. 4. 2024 20:34 uživatel shocker990 ***@***.***>
napsal:

**Comment by felipeliliti** [comment]
Projeto liliti stk 3.6.9 inteligência artificial multimidal só para passar um nível aqui beta. Kkkkkk fuii

**Reply by grandmas-favorite** [reply]
Hi @felipeliliti, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting.

**Reply by felipeliliti** [reply]
Ok meu broder preciso de ajuda financeira como faço?

**Comment by felipeliliti** [comment]
No problem, I understand. Disponibiliza para português uai? Kkkk

Em dom, 5 de mai de 2024 18:41, grandmas-favorite ***@***.***>
escreveu:

**Comment by sardorml** [comment]
This is happening to me as well. I am using Firefox browser.
